src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ClusterBox.tsx

```diff
@@ -0,0 +1,86 @@
+// Libraries
+import React, {FC, useCallback, useContext, useMemo} from 'react'
+import classnames from 'classnames'
+import {FlexBox, FlexDirection} from '@influxdata/clockface'
+
+// Style
+import './ClusterBox.scss'
+
+// Components
+import {CreateOrgContext, ProviderID} from './CreateOrganizationContext'
+import {Cluster} from 'src/client/unityRoutes'
+import {AWSLogo} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AWSLogo'
+import {AzureLogo} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AzureLogo'
+import {GCPLogo} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/GCPLogo'
+import {AzureLogoWithText} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AzureLogoWithText'
+import {GCPLogoWithText} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/GCPLogoWithText'
+import {AWSLogoColor} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AWSLogoColor'
+
```

Comment:
On balance I would alphabetize the component imports so that they're easier to find, but I see the logic behind how you've organized them here.